libpod/networking_slirp4netns.go

```diff
@@ -391,6 +391,16 @@ func (r *Runtime) setupRootlessPortMappingViaRLK(ctr *Container, netnsPath strin
 	}
 
 	childIP := slirp4netnsIP
```

Comment:
Can we remove this const `slirp4netnsIP = "10.0.2.100"`?

Reply:
If the slirp cidr is omitted, we still need the default ip.

Reply:
If we have the default CIDR string (10.0.2.0/24), we no longer need the default IP string.

Reply:
OK but than we just changed the ip with a subnet.